Makefile

```diff
@@ -212,7 +212,7 @@ can-build-pr:
 i18n-extract: ## Extract strings for translation from the source code
 	npm run mmjstool -- i18n extract-mobile
 
-i18n-extract-ci:
+i18n-checks:
```

Comment:
I think we need both independently, one for checks the other one for extraction.. I can see them used  in two different use cases